tune_sklearn/tune_search.py

```diff
@@ -626,6 +626,8 @@ def _tune_run(self, config, resources_per_trial):
             resources_per_trial (dict): Resources to use per trial within Ray.
                 Accepted keys are `cpu`, `gpu` and custom resources, and values
                 are integers specifying the number of each resource to use.
+            tune_params (dict, optional): User defined parameters passed to
```

Comment:
I think to get this to properly render on readthedocs, you can just leave this as (dict)

Reply:
Can you add a comment here also about the overriding behavior?